src/eos-autoupdater.c

```diff
@@ -30,13 +50,17 @@ typedef enum _UpdateStep {
   UPDATE_STEP_APPLY
 } UpdateStep;
 
+/* These must be kept in sync with #UpdateStep. */
+#define UPDATE_STEP_FIRST UPDATE_STEP_NONE
+#define UPDATE_STEP_LAST UPDATE_STEP_APPLY
+
 #define SEC_PER_DAY (3600ll * 24)
 
 /* This file is touched whenever the updater starts */
-#define UPDATE_STAMP_DIR        "/var/lib/eos-updater"
-#define UPDATE_STAMP_NAME       "eos-updater-stamp"
+static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
```

Comment:
Nope: `debian/rules` passes `--localstatedir=/var/lib` to `configure`. IIRC this is the ‘other’ (non-GNU) default value for `localstatedir`. But if you can point me to a good precedent, I can easily move the `/lib` from `debian/rules` to `eos-autoupdater.c`.